src/main.ts

```diff
@@ -1,59 +1,105 @@
-import * as light from '@lightprotocol/zk.js'
+import * as light from "@lightprotocol/zk.js";
 import * as anchor from "@coral-xyz/anchor";
-import { Account, airdropSol, confirmConfig, LOOK_UP_TABLE, TestRelayer, User } from '@lightprotocol/zk.js';
+import {
+  airdropSol,
+  confirmConfig,
+  LOOK_UP_TABLE,
+  TestRelayer,
+  User,
+} from "@lightprotocol/zk.js";
+const SOLANA_PORT = process.env.SOLANA_PORT || "8899";
 process.env.ANCHOR_WALLET = process.env.HOME + "/.config/solana/id.json";
-process.env.ANCHOR_PROVIDER_URL = "http://127.0.0.1:8899";
+process.env.ANCHOR_PROVIDER_URL = `http://127.0.0.1:${SOLANA_PORT}`;
```

Comment:
not sure that this makes sense because the test relayer relies on the local test validator to run on this port

Reply:
sure, I can revert it. I am just running over a "versatile" way of coding :)